modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/protocol/v0.1.0.json

```diff
@@ -29,14 +28,15 @@
           "path": "schema:identifier",
           "name": "Identifier",
           "description": "The identifier of protocol such as a DOI, ISBN,...",
-          "node": "this:IdentifierShape",
+          "datatype": "xsd:anyURI",
```

Comment:
Not every protocol might have a URI (e.g. if we only get a pdf of the protocol). Maybe we should remove the minCount on the identifier property?

Reply:
Okay. Removed mincount and maxcount.